src/lib/wallet/MultipleHttpProvider.js

```diff
@@ -52,8 +53,15 @@ export class MultipleHttpProvider extends HttpProvider {
         if (isConnectionError(exception) && !loggedProviders.has(provider)) {
           loggedProviders.set(provider, true)
 
+          const { message: originalMessage } = exception
+          exception.message = 'Failed to connect RPC'
```

Comment:
why are you overwriting the message?

Reply:
The aggregation of the network errors on sentry you asked for